modules/mono/mono_gd/gd_mono_class.cpp

```diff
@@ -340,11 +340,11 @@ const Vector<GDMonoField *> &GDMonoClass::get_all_fields() {
 		Map<StringName, GDMonoField *>::Element *match = fields.find(name);
 
 		if (match) {
-			fields_list.push_back(match->get());
+			fields_list.insert(0, match->get());
 		} else {
 			GDMonoField *field = memnew(GDMonoField(raw_field, this));
 			fields.insert(name, field);
-			fields_list.push_back(field);
+			fields_list.insert(0, field);
```

Comment:
I'd prefer to add it in the right order here, and iterate in reverse order in _update_exports, but it's still fine :)

Reply:
No problem, I'll take a look.